test/unit/app/views/extends.nunjucks

```diff
@@ -0,0 +1,4 @@
+{% extends 'subview.master' %}
```

Comment:
May be `subviews.master`. It failed https://travis-ci.org/adonisjs/adonis-framework/jobs/126437887#L1259